src/core/patch.rkt

```diff
@@ -91,13 +100,14 @@
   (timeline-push! 'inputs (map ~a altns))
 
   (define approx->prev (make-hasheq))
+
   (define approxs
     (reap [sow]
-          (for ([altn (in-list altns)])
-            (for ([approximation (taylor-alt altn)])
-              (unless (spec-has-nan? (alt-expr approximation))
-                (hash-set! approx->prev approximation altn)
-                (sow approximation))))))
+          (for ([approximation (taylor-alts altns)])
+            (unless (spec-has-nan? (alt-expr approximation))
+              ; here (car (alt-prevs approximation)) is simply an original altn
+              (hash-set! approx->prev approximation (car (alt-prevs approximation)))
+              (sow approximation)))))
```

Comment:
As a result we have the same `approxs`, but in a different order. 
My assumption is that this introduces changes in output accuracy on a nightly run.
The fact that the `approxs` contains the same items as before was checked